homeassistant/components/overkiz/config_flow.py

```diff
@@ -48,21 +48,11 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    _config_entry: ConfigEntry | None
-    _api_type: APIType
-    _user: None | str
-    _server: str
-    _host: str
-
-    def __init__(self) -> None:
-        """Initialize Overkiz Config Flow."""
-        super().__init__()
-
-        self._config_entry = None
-        self._api_type = APIType.CLOUD
-        self._user = None
-        self._server = DEFAULT_SERVER
-        self._host = "gateway-xxxx-xxxx-xxxx.local:8443"
+    reauth_entry: ConfigEntry | None = None
```

Comment:
@joostlek why would reauth_entry be without `_`? I understood that prefixing it with underscore hints that it is for internal use / private variable. Don't see reauth_entry being used outside of this class?

Reply:
Oh whoops, that slipped